src/ui/layout/Layout.vue

```diff
@@ -289,14 +292,19 @@ export default {
             this.triggerReset = !this.triggerReset;
         },
         handleClearRecentObjects() {
+            this.disableClearButton = true;
             this.$refs.recentObjectsList.clearRecentObjects();
         },
         onStartResizing() {
             this.isResizing = true;
         },
         onEndResizing() {
             this.isResizing = false;
+        },
+        handleRecentObjectsListUpdated() {
```

Comment:
Two things:
- Let's rename this to `setClearButtonDisabled`
- Let's update this to receive the emitted boolean value and assign it to `this.disableClearButton`